web/src/Home.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
```

Comment:
Can you move the components to a components folder, so we have:

```
src
|__components/
     |___ ...
|__ App.js
```

ping me on discord if you want to do this differently but im not a fan of just having everything being in src unorganized

Reply:
added all of the new files to components good call!